tests/integration/golang/helpers/gs.go

```diff
@@ -25,10 +25,18 @@ func NewGSClient(endpoint string) (*storage.Client, error) {
 	return client, nil
 }
 
-// PrepareTestBuckets prepares test buckets for further usage in tests:
-// - delete provided buckets and all the content within them.
-// - create the same empty bucket.
-func PrepareTestBuckets(client *storage.Client, buckets []string) error {
+// CreateGSBuckets creates tests buckets in the Google Storage.
+func CreateGSBuckets(client *storage.Client, buckets []string) error {
```

Comment:
I really like that you're passing the buckets here instead of having some global variable — could you please modify the S3 tests to do the same thing? (in another PR, of course)

Reply:
sure!

Reply:
@jgiannuzzi addressed #509